src/app/about/presentation/presentation.component.scss

```diff
@@ -5,48 +5,41 @@
 @use 'page';
 
 :host {
-  @include page.full-size;
```

Comment:
Moved to about component. And simplified by applying `.contents` to `:host` directly